.slim-lint.yml

```diff
@@ -0,0 +1,6 @@
+exclude:
+  - "**/lib/templates/slim/**/*"
```

Comment:
These are actually ERB templates that generate slim files. The linter is not capable of parsing them.